pkg/products/fuse/reconciler.go

```diff
@@ -293,42 +292,15 @@ func (r *Reconciler) reconcileCloudResources(ctx context.Context, rhmi *integrea
 		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("failed to reconcile postgres instance for fuse: %w", err)
 	}
 
-	// create prometheus failed rule
-	_, err = resources.CreatePostgresResourceStatusPhaseFailedAlert(ctx, client, rhmi, postgres)
+	// reconcile postgres alerts
+	phase, err := resources.ReconcilePostgresAlerts(ctx, client, rhmi, postgres)
 	if err != nil {
-		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("failed to create postgres failure alert for fuse: %w", err)
+		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("failed to reconcile postgres alerts: %w", err)
```

Comment:
minor nit but maybe it would be handy to have the product name in the error message just for observability? Most of the old errors being removed were along the lines of `"failed to create postgres failure alert for fuse: <message>"` perhaps we should do that for all of the `ReconcilePostgresAlerts` calls made in each product reconciler.